daemon/mgr/cri.go

```diff
@@ -38,10 +65,86 @@ func (c *CriManager) Version(ctx context.Context, r *runtime.VersionRequest) (*r
 	return nil, fmt.Errorf("Version Not Implemented Yet")
 }
 
+func makeLabels(labels, annotations map[string]string) map[string]string {
+	m := make(map[string]string)
+
+	for k, v := range labels {
+		m[k] = v
+	}
+	for k, v := range annotations {
+		// Use prefix to distinguish between annotations and labels.
+		m[fmt.Sprintf("%s%s", annotationPrefix, k)] = v
+	}
+
+	return m
+}
+
+// makeSandboxPouchConfig returns apitypes.ContainerCreateConfig based on runtimeapi.PodSandboxConfig.
+func (c *CriManager) makeSandboxPouchConfig(config *runtime.PodSandboxConfig, image string) (*apitypes.ContainerCreateConfig, error) {
+	// Merge annotations and labels because pouch supports only labels.
+	labels := makeLabels(config.GetLabels(), config.GetAnnotations())
+	// Apply a label to distinguish sandboxes from regular containers.
+	labels[containerTypeLabelKey] = containerTypeLabelSandbox
+
+	hc := &apitypes.HostConfig{}
```

Comment:
If https://github.com/alibaba/pouch/pull/459 has been merged, this part needs change, since we should make NetworkingConfig non-nil.

Reply:
OK, please ping me if it is merged :)

Reply:
get merged, please rebase, thanks

Reply:
@allencloud  updated.